package.json

```diff
@@ -13,53 +13,70 @@
     "type": "git",
     "url": "https://github.com/brightcove/pkcs7"
   },
-  "license": "Apache2",
+  "license": "Apache-2.0",
   "files": [
-    "bin",
-    "src",
-    "dist"
+    "CONTRIBUTING.md",
+    "dist/",
+    "docs/",
+    "index.html",
+    "scripts/",
+    "src/",
+    "test/"
   ],
   "keywords": [
     "pkcs7",
-    "pkcs7 encrypt decrypt padding pkcs"
+    "pkcs7 encrypt decrypt padding pkcs",
+    "videojs",
+    "videojs-plugin"
   ],
+  "dependencies": {
+    "global": "^4.3.2"
+  },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-es3": "^1.0.1",
-    "mkdirp": "^0.5.1",
-    "nodeunit": "^0.11.0",
-    "npm-run-all": "^4.0.2",
-    "rimraf": "^2.6.1",
-    "rollup": "^0.41.6",
-    "rollup-plugin-babel": "^2.7.1",
-    "rollup-plugin-json": "^2.1.1",
-    "rollup-watch": "^3.2.2",
-    "videojs-standard": "^5.2.0"
+    "conventional-changelog-cli": "^2.0.1",
+    "conventional-changelog-videojs": "^3.0.0",
+    "doctoc": "^1.3.1",
+    "husky": "^1.0.0-rc.13",
+    "in-publish": "^2.0.0",
+    "jsdoc": "https://github.com/BrandonOCasey/jsdoc#feat/plugin-from-cli",
+    "karma": "^3.0.0",
+    "lint-staged": "^7.2.2",
+    "nodeunit": "^0.11.3",
+    "not-prerelease": "^1.0.1",
+    "npm-merge-driver-install": "^1.0.0",
+    "npm-run-all": "^4.1.3",
+    "pkg-ok": "^2.2.0",
+    "rollup": "^0.65.0",
+    "shx": "^0.3.2",
+    "sinon": "^6.1.5",
+    "videojs-generate-karma-config": "~3.0.0",
+    "videojs-generate-rollup-config": "~2.2.0",
+    "videojs-standard": "^7.1.0"
   },
   "scripts": {
     "prebuild": "npm run clean",
     "build": "npm-run-all -p build:*",
-    "build:js": "npm-run-all -p build:js:modules build:js:umd build:js:pad build:js:unpad",
-    "build:js:modules": "rollup -c scripts/modules.rollup.js",
-    "build:js:umd": "rollup -c scripts/umd.rollup.js",
-    "build:js:pad": "rollup -c scripts/pad.rollup.js",
-    "build:js:unpad": "rollup -c scripts/unpad.rollup.js",
-    "build:test": "rollup -c scripts/test.rollup.js",
-    "clean": "rimraf dist test/dist",
-    "postclean": "mkdirp dist test/dist",
+    "build:js": "rollup -c scripts/rollup.config.js",
+    "clean": "shx rm -rf ./dist ./test/dist",
+    "postclean": "shx mkdir -p ./dist ./test/dist",
+    "docs": "npm-run-all docs:*",
+    "docs:api": "jsdoc src -g plugins/markdown -r -d docs/api",
+    "docs:toc": "doctoc README.md",
     "lint": "vjsstandard",
-    "pretest": "npm-run-all lint build:test",
-    "test": "nodeunit test/dist/pkcs7_test.js",
+    "server": "karma start scripts/karma.conf.js --singleRun=false --auto-watch",
+    "start": "npm-run-all -p server watch",
+    "pretest": "npm-run-all lint build",
+    "test": "npm-run-all -p test:browser test:node",
+    "test:browser": "karma start scripts/karma.conf.js",
+    "test:node": "qunit test/dist/bundle.js",
```

Comment:
Previously tests only ran on node with `nodeunit` but `qunit` can now run its tests in node. So after switching the tests to `QUnit` we run them in both places. Super cool.